raiden/network/transport.py

```diff
@@ -81,6 +87,9 @@ def __init__(
     def receive(self, data, host_port):  # pylint: disable=unused-argument
         try:
             self.protocol.receive(data)
+        except InvalidProtocolMessage as e:
```

Comment:
Why do we need that here?

Reply:
An alternative is the protocol receive:

https://github.com/raiden-network/raiden/blob/master/raiden/network/protocol.py#L822-L863

Please, guard the `log.warning` with the `isEnabledFor` check

Reply:
@hackaugusto `isEnabledFor()` is there just to avoid string concat? That's the one of the ugliest thing's I've seen in Python :)

Reply:
Yes, it's to avoid the logging infrastructure on the hot path. We benchmarked a few months back and the performance impact is significant.

I too dislike it, suggestions are welcome, but for the moment lets stick with the same pattern that is used throughout the rest of the code base.

Reply:
Yeah, I see the point and I'll adapt the code accordingly. But does it really matter to use this idiom in paths which occur only rarely? This exception is unlikely going to be raised and cause a performance issue, unless someone is DDoS'ing the node.